### ISSUE TITLE
Standup Cumulus in CBA Prod account

### ISSUE THREAD

**Issue by chuckwondo** [NASA-IMPACT/csdap-cumulus#127]
Do initial Cumulus deployment to `csda-cumulus-prod-5047`. This task does not include any effort to get the deployment fully functional, but rather, only the successful creation of resources managed by Terraform.

1. If not already created, create a `.env.cba-prod` file with the following contents:
    ```
    AWS_REGION=us-west-2
    AWS_PROFILE=csda-cumulus-prod-5047
    TS_ENV=prod
    ```
1. If not already created, create an AWS profile named `csda-cumulus-prod-5047` using AWS long-term access keys generated in Kion
1. Run the following (the first command might take 10 minutes or so, and the second might take 2 hours or so):
    ```
    DOTENV=.env.cba-prod make pre-deploy-setup
    DOTENV=.env.cba-prod make all-up-yes
    ```
1. If the second command fails, rerun it, as it might simply fail due to a Terraform race condition. The second time should succeed.

**Comment by chuckwondo** [comment]
CBA Prod's initial deployment is mostly complete:

- rds-cluster module fully deployed
- data-persistence module fully deployed
- cumulus module mostly deployed

The cumulus module deployment failed with a number of the following errors:

```plain
AccessDeniedException: User with accountId: ********5047 is not authorized to perform: logs:PutSubscriptionFilter on resource: arn:aws:logs:us-west-2:********7126:destination:CumulusRecipient-LogDestinationWest
```

This indicates that the Metrics team must give our CBA Prod account permission to subscribe to the log destination.

I have asked Metrics to add the necessary permission on their side, and Amanda Monteavaro responded, indicating that she will update it later today (Nov. 9, 2023). Once updated, I'll retry deploying the cumulus module and follow up with a comment indicating results (success or more errors).

**Comment by chuckwondo** [comment]
The Metrics team added the missing permissions on their side, and now there's only 1 remaining deployment error related to the policy for the system bucket. I've fixed the issue in the PR: https://github.com/NASA-IMPACT/csdap-cumulus/pull/291

In addition, that PR also re-enables deployment to prod, which will now go to CBA Prod, because I updated the `AWS_ROLE_TO_ASSUME_ARN` secret for the [github prod environment](https://github.com/NASA-IMPACT/csdap-cumulus/settings/environments/539992441/edit) to the ARN of the github-oidc role in CBA Prod.

Once the PR is approved and merged, it will deploy to CBA UAT, and then to CBA Prod (after manual approval).
